packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -375,11 +376,15 @@ export default [
         const remotes =
           job.remotes !== undefined ? destructPattern(job.remotes) : []
         const srs = job.srs !== undefined ? destructPattern(job.srs) : []
+        const { concurrency, offlineSnapshot, timeout } = job.settings['']
```

Comment:
Are we sure `job.settings['']` is defined here?

Reply:
Better use 
`const displayAdvancedSettings = defined(state.displayAdvancedSettings,  concurrency > 0 || timeout > 0 || offlineSnapshot)` 
in the render.

Reply:
It's fix.